src/tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "removeComments": false,
-    "noImplicitAny": false,
+    "noImplicitAny": true,
```

Comment:
Yeah, I probably should have put this as a separate commit instead of trying to sneak it in along with this changeset, but yeah I also think it's a good thing to have this enabled.